lib/puppet/version.rb

```diff
@@ -73,19 +67,19 @@ def self.version
     @puppet_version ||= PUPPETVERSION
   end
 
-  # @return [SemanticPuppet::Version] the puppet version
-  def self.parsed_version
-    @parsed_version ||= SemanticPuppet::Version.parse(version)
+  # @return [String] containing the puppet major version, e.g. "3"
+  def self.version_major
```

Comment:
I liked the old name `major_version` better. At the same time, I think this PR would be fine if this method was removed altogether. AFAIK, no one has requested a method that returns the major version number in string form.